cmd/expired-authz-purger/main.go

```diff
@@ -168,19 +193,20 @@ func main() {
 	err = features.Set(config.ExpiredAuthzPurger.Features)
 	cmd.FailOnError(err, "Failed to set feature flags")
 
-	auditlogger := cmd.NewLogger(config.ExpiredAuthzPurger.Syslog)
-	auditlogger.Info(cmd.VersionString())
+	logger := cmd.NewLogger(config.ExpiredAuthzPurger.Syslog)
+	logger.Info(cmd.VersionString())
 
-	defer auditlogger.AuditPanic()
+	defer logger.AuditPanic()
 
 	// Configure DB
 	dbURL, err := config.ExpiredAuthzPurger.DBConfig.URL()
 	cmd.FailOnError(err, "Couldn't load DB URL")
 	dbMap, err := sa.NewDbMap(dbURL, config.ExpiredAuthzPurger.DBConfig.MaxDBConns)
 	cmd.FailOnError(err, "Could not connect to database")
+	sa.SetSQLDebug(dbMap, logger)
```

Comment:
Do we want to keep this?

Reply:
Yeah, I think it's useful. It doesn't show up with stdoutLevel: 6, so it's harmless, but it's nice to have if we want to see more detail.